src/const.js

```diff
@@ -58,3 +57,11 @@ export const METADATA_TIME_KEY = 'generationTime';
 export const UNSUPPORTED_TYPE = 'Unsupported Data Type';
 export const AGGREGATE_TYPE = 'AGGREGATE';
 export const NAMESPACE = 'taxonomy';
+
+export const MDB_OBJECT = Object.freeze({
```

Comment:
`Object.freeze` is to stop it being reactified by Vue?

Reply:
No, since we assign this to a class instance (this.MDB_OBJECT), it's mostly to make sure nothing changes it's type.

Reply:
Oh I see, makes the object a true const :) Nice!